src/rt/test/func/external_threading/build_test.cc

```diff
@@ -0,0 +1,13 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+#define VERONA_EXTERNAL_THREADING
+#include <verona.h>
+
+/**
+ * This test if only a build test to confirm that the template of
+ * verona_external_threading.h matches the requirements of the Verona run-time.
+ */
+int main()
+{
+  return 0;
```

Comment:
Mildly concerned that template instantiation could hide compile bugs as this doesn't actually call the runtime.